theseus_gui/src/pages/project/Index.vue

```diff
@@ -0,0 +1,356 @@
+          </a>
+          <a
+              v-if="data.source_url"
+              :href="data.source_url"
+              class="title"
+              rel="noopener nofollow ugc"
+          >
+            <CodeIcon aria-hidden="true" />
+            <span>Source</span>
+          </a>
+          <a
+              v-if="data.wiki_url"
+              :href="data.wiki_url"
+              class="title"
+              rel="noopener nofollow ugc"
+          >
+            <WikiIcon aria-hidden="true" />
+            <span>Wiki</span>
+          </a>
+          <a
+              v-if="data.wiki_url"
+              :href="data.wiki_url"
+              class="title"
+              rel="noopener nofollow ugc"
+          >
+            <DiscordIcon aria-hidden="true" />
+            <span>Discord</span>
+          </a>
+          <a
+              v-for="(donation, index) in data.donation_urls"
+              :key="index"
+              :href="donation.url"
+              rel="noopener nofollow ugc"
+          >
+            <BuyMeACoffeeIcon v-if="donation.id === 'bmac'" aria-hidden="true" />
+            <PatreonIcon v-else-if="donation.id === 'patreon'" aria-hidden="true" />
+            <KoFiIcon v-else-if="donation.id === 'ko-fi'" aria-hidden="true" />
+            <PaypalIcon v-else-if="donation.id === 'paypal'" aria-hidden="true" />
+            <OpenCollectiveIcon
+                v-else-if="donation.id === 'open-collective'"
+                aria-hidden="true"
+            />
+            <HeartIcon v-else-if="donation.id === 'github'" />
+            <CoinsIcon v-else />
+            <span v-if="donation.id === 'bmac'">Buy Me a Coffee</span>
+            <span v-else-if="donation.id === 'patreon'">Patreon</span>
+            <span v-else-if="donation.id === 'paypal'">PayPal</span>
+            <span v-else-if="donation.id === 'ko-fi'">Ko-fi</span>
+            <span v-else-if="donation.id === 'github'">GitHub Sponsors</span>
+            <span v-else>Donate</span>
+          </a>
+        </div>
+      </Card>
+    </div>
+    <div v-if="data" class="content-container">
+      <Promotion />
+      <Card class="tabs">
+        <NavRow
+          v-if="data.gallery.length > 0"
+          :links="[
+            {
+              label: 'Description',
+              href: `/project/${$route.params.id}/`,
+            },
+            {
+              label: 'Versions',
+              href: `/project/${$route.params.id}/versions`,
+            },
+            {
+              label: 'Gallery',
+              href: `/project/${$route.params.id}/gallery`,
+            }
+          ]"
+        />
+        <NavRow
+          v-else
+          :links="[
+            {
+              label: 'Description',
+              href: `/project/${$route.params.id}/`,
+            },
+            {
+              label: 'Versions',
+              href: `/project/${$route.params.id}/versions`,
+            }
+          ]"
+        />
+      </Card>
+      <RouterView/>
```

Comment:
The data fetching overall is kind of weird and makes the UX weird as well. It should use the created hook for fetching.

Instead of fetching each component individually, I would just fetch everything in the projects parent page (see how the frontend does it here: https://github.com/modrinth/knossos/blob/master/pages/%5Btype%5D/%5Bid%5D.vue#L739-L779) and then pass it using props to the child components. RIght now it feels slower than it needs to be and a little wasteful, like it fetches the project twice to get the description and the body